R/civis_ml_utils.R

```diff
@@ -19,19 +19,93 @@ CIVIS_ML_CLASSIFIERS <- c("sparse_logistic",
                           "multilayer_perceptron_classifier",
                           "stacking_classifier")
 
-CIVIS_ML_TEMPLATE_IDS <- data.frame(
-  id = c(9112, 9113, 9968, 9969, 10582, 10583, 11219, 11220, 11221),
-  version = c(1.1, 1.1, 2.0, 2.0, 2.1, 2.1, 2.2, 2.2, 2.2),
-  name = c("train", "predict", "train", "predict", "train", "predict", "train", "predict", "register"),
-  stringsAsFactors = FALSE
-)
+
+get_template_ids_all_versions <- function(){
```

Comment:
Should we write tests for this new function? In civis-python, this is what I did:

https://github.com/civisanalytics/civis-python/pull/341/files#diff-945a68cf4837987241a1965da63ad812R721-R726

Reply:
I think we should take this opportunity to refactor global package state (including options) and just call the function directly when we need it.

Reply:
Sorry I'm not sure what you mean by "global package state (including options) we can just call the function directly when we need it" I tried to read up on it but couldn't find a specific part of http://r-pkgs.had.co.nz/ to read? Would you be able to point me to where I can learn more about this to make this change? Thanks!